js/common/lib/tensor.ts

```diff
@@ -132,15 +139,27 @@ export declare namespace Tensor {
    */
   export type GpuBufferType = {size: number; mapState: 'unmapped' | 'pending' | 'mapped'};
 
+  /**
+   * type alias for WebNN MLBuffer
+   *
+   * The specification for WebNN's ML Buffer is currently in flux.
+   */
+  export type MlBufferType = unknown;
```

Comment:
naming:

considering webnn spec, it is recommended to use `mlBuffer`(variable/property) or `MLBuffer`(type/interface), but not `MlBuffer`.

Reply:
Renamed all `MlBuffer`s and `MlContext`s to match the WebNN spec.